python/ray/client_builder.py

```diff
@@ -96,6 +96,8 @@ def _get_builder_from_address(address: Optional[str]) -> ClientBuilder:
         return _LocalClientBuilder(address)
     module_string, inner_address = _split_address(address)
     module = importlib.import_module(module_string)
```

Comment:
You might also need to try except this importlib.import_module()

Reply:
Definitely!